src/os_abstract.h

```diff
@@ -136,7 +137,8 @@ void* win_set_timer(void *p_timer);
 #include <unistd.h>
 #include <sys/syscall.h>
 #include <endian.h>
-
+#define SP_FORCE_INLINE inline __attribute__ ((always_inline))
+#define THREAD_LOCAL	__thread
```

Comment:
Did you verify that your code compiles in all supported platforms and not only gcc and windows (for example BSD)?